builder/oracle/oci/config.go

```diff
@@ -169,6 +173,30 @@ func NewConfig(raws ...interface{}) (*Config, error) {
 			errs, errors.New("'base_image_ocid' must be specified"))
 	}
 
+	// Validate tag lengths. TODO (hlowndes) maximum number of tags allowed.
```

Comment:
It looks like this TODO can be removed, since you have the check against a length of 100, is that right?

Reply:
@SwampDragons the length is for the length of the tag key/value. The TODO refers to the number of tags allowed on a single OCI resource.